src/CoreBundle/private/library/classes/TCMSLogChange.class.php

```diff
@@ -1779,6 +1779,51 @@ public static function escapeJSOutput($str)
         return $str;
     }
 
+    /**
+     * This method lets old updates insert shop system pages. The table "shop_system_page" has been removed in shop-165.inc.php and
+     * old updates would fail when triggered afterwards.
+     *
+     * You should not use this method in new packages, which already know about the change.
+     *
+     * @param $shop_id
+     * @param $name_internal
+     * @param $name
+     * @param string $cms_tree_id
+     *
+     * @deprecated since 6.2.0 - no longer used.
+     */
+    public static function addShopSystemPage($shop_id, $name_internal, $name, $cms_tree_id = '', $id = null)
```

Comment:
Where is this/will this be used?

Reply:
Is it used by dependent code outside of the Chameleon core system.